account_banking_payment_export/models/account_payment.py

```diff
@@ -35,6 +35,12 @@ class PaymentOrder(models.Model):
         readonly=True, states={'draft': [('readonly', False)]})
     mode_type = fields.Many2one('payment.mode.type', related='mode.type',
                                 string='Payment Type')
+    total = fields.Float(compute='_total', store=True)
+
+    @api.depends('line_ids', 'line_ids.amount')
+    @api.one
+    def _total(self):
```

Comment:
I called it this way in order to be sure it overrides the method used for the functional field defined in the core module `account_payment`. Do you think we can define another instead of overriding this one ?